Source/UI/SecurityCodeComponent/SecurityCodeComponent.swift

```diff
@@ -0,0 +1,56 @@
+// 
+//  SecurityCodeComponent.swift
+//  
+//
+//  Created by Okhan Okbay on 26/09/2023.
+//
+
+import Checkout
+import UIKit
+
+public final class SecurityCodeComponent: UIView {
```

Comment:
Android calls it `CVVComponent` Please check if you want to sync this up accordingly. CC @chintan-soni-cko

Reply:
It was as security code on iOS so decided to keep it as before but will re-review

Reply:
It would be nice if we were almost complete with the spike work for IOS and [updating api spec](https://checkout.atlassian.net/wiki/spaces/FS/pages/5855281337/Draft+CVV+Tokenisation+API+Spec). So, we can easily align across both platform and address it by reviewing it. Just a thought, feel free how okhan you would like to sync up for it.

Reply:
yeah this PR was to review the code and hold light to update API Spec. Now will do it 🙂

Reply:
We discussed this offline and we're going forward with `Security Code` wording and derivations thereof on iOS only